spreadsheets/collision_rank.yaml

```diff
@@ -0,0 +1,943 @@
+- {layer: 'pois', kind: 'harbourmaster'}
+- {layer: 'pois', kind: 'helipad'}
+- {layer: 'pois', kind: 'heliport'}
+- {layer: 'pois', kind: 'hot_spring'}
+- {layer: 'pois', kind: 'hunting'}
+- {layer: 'pois', kind: 'information'}
+- {layer: 'earth', kind: 'islet'}
+- {layer: 'pois', kind: 'mini_roundabout'}
+- {layer: 'pois', kind: 'miniature_golf'}
+- {layer: 'pois', kind: 'photo'}
+- {layer: 'pois', kind: 'picnic_site'}
+- {layer: 'pois', kind: 'snow_cannon'}
+- {layer: 'pois', kind: 'swimming_area'}
+- {layer: 'pois', kind: 'tyres'}
+- {layer: 'pois', kind: 'block'}
+- {layer: 'pois', kind: 'bunker'}
+# reserve for kind_details
+- _reserved: {from: 3178, to: 3198}
+- {layer: 'pois', kind: 'shelter'}
+- {layer: 'pois', kind: 'recreation_track'}
+- {layer: 'pois', kind: 'recycling'}
+- {layer: 'pois', kind: 'adit'}
+- {layer: 'pois', kind: 'allotments'}
+- {layer: 'pois', kind: 'archaeological_site'}
+- {layer: 'pois', kind: 'bicycle_junction'}
+- {layer: 'pois', kind: 'boat_lift'}
+- {layer: 'pois', kind: 'pitch'}
+# reserve for kind_details
+- _reserved: {from: 3208, to: 3308}
+- {layer: 'pois', kind: 'bollard'}
+- {layer: 'pois', kind: 'cattle_grid'}
+- {layer: 'pois', kind: 'crane'}
+# reserve for kind_details
+- _reserved: {from: 3312, to: 3322}
+- {layer: 'pois', kind: 'ford'}
+- {layer: 'pois', kind: 'halt'}
+- {layer: 'pois', kind: 'plaque'}
+- {layer: 'pois', kind: 'power_tower'}
+- {layer: 'pois', kind: 'stop'}
+- {layer: 'pois', kind: 'tram_stop'}
+- {layer: 'pois', kind: 'tree'}
+- {layer: 'pois', kind: 'walking_junction'}
+- {layer: 'pois', kind: 'wayside_cross'}
+- {layer: 'landuse', kind: 'taxiway'}
+- {layer: 'landuse', kind: 'runway'}
+- {layer: 'landuse', kind: 'wastewater_plant'}
+- {layer: 'landuse', kind: 'water_works'}
+- {layer: 'landuse', kind: 'works'}
+- {layer: 'landuse', kind: 'enclosure'}
+- {layer: 'landuse', kind: 'roller_coaster'}
+- {layer: 'landuse', kind: 'allotments'}
+- {layer: 'landuse', kind: 'generator'}
+- {layer: 'landuse', kind: 'maze'}
+- {layer: 'landuse', kind: 'hanami'}
+- {layer: 'landuse', kind: 'amusement_ride'}
+- {layer: 'landuse', kind: 'picnic_site'}
+- {layer: 'landuse', kind: 'crane'}
+# reserve for kind_details
+- _reserved: {from: 3346, to: 3356}
+- {layer: 'landuse', kind: 'embankment'}
+- {layer: 'transit', kind: 'halt'}
+- {layer: 'transit', kind: 'stop'}
+- {layer: 'transit', kind: 'tram_stop'}
+- {layer: 'pois', kind: 'accountant'}
+- {layer: 'pois', kind: 'administrative'}
+- {layer: 'pois', kind: 'advertising_agency'}
+- {layer: 'pois', kind: 'ambulatory_care'}
+# reserve for kind_details
+- _reserved: {from: 3365, to: 3385}
+- {layer: 'pois', kind: 'architect'}
+- {layer: 'pois', kind: 'art'}
+- {layer: 'pois', kind: 'association'}
+- {layer: 'pois', kind: 'aviary'}
+- {layer: 'pois', kind: 'baby_hatch'}
+- {layer: 'pois', kind: 'bakery'}
+- {layer: 'pois', kind: 'beauty'}
+- {layer: 'pois', kind: 'bed_and_breakfast'}
+- {layer: 'pois', kind: 'blood'}
```

Comment:
Remove `blood`, it was a jumbled version of `blood_bank`, which we have immediately below.

Reply:
Fixed in cfe0df9.